server/container_events.go

```diff
@@ -0,0 +1,17 @@
+package server
+
+import (
+	types "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
+// GetContainerEvents sends the stream of container events to the client - kubelet
```

Comment:
while we primarily care about the kubelet it's not necessarily the only client. there may be a `crictl events` call one day